src/main/java/com/blockout/View.java

```diff
@@ -1,6 +1,7 @@
 package com.blockout;
 
 import com.blockout.views.Window;
+import net.minecraft.client.renderer.GlStateManager;
 import org.lwjgl.opengl.GL11;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.lwjgl.opengl.GL11'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)